### PR TITLE
fix hardcoded request type

### PR DESCRIPTION
**Pull request by annapendleton**
Small fix for hardcoded request type in locust - plumbs variable through and adds extra logging in run.sh.

Currently under e2e validation

### CODE REVIEW

File: benchmarks/benchmark/tools/locust-load-inference/sample-terraform.tfvars [hunk 1]
```diff
@@ -9,7 +9,7 @@ ksa       = "benchmark-ksa"
 
 # Locust service configuration 
 artifact_registry                        = "us-central1-docker.pkg.dev/$PROJECT_ID/ai-benchmark"
-inference_server_service                 = "tgi" # inference server service name
+inference_server_service                 = "http://tgi" # inference server service name
```
Comment (hubatish) [inline benchmarks/benchmark/tools/locust-load-inference/sample-terraform.tfvars:12]: This struck me as weird. Glad it's being changed.

File: benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/run.sh [hunk 2]
```diff
@@ -35,7 +37,10 @@ if [[ "$LOCUST_MODE" = "master" ]]; then
         LOCUST_OPTS="$LOCUST_OPTS --stop-timeout $STOP_TIMEOUT"
     fi
 elif [[ "$LOCUST_MODE" = "worker" ]]; then
+
+    echo "login to huggingface"
     huggingface-cli login --token $HUGGINGFACE_TOKEN
```
Comment (hubatish) [inline benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/run.sh:42]: Does the login have any existing output? Is it possible to log something helpful on login failure?

File: benchmarks/benchmark/tools/locust-load-inference/sample-terraform.tfvars (hunk 1 above)
Comment (kfswain) [inline benchmarks/benchmark/tools/locust-load-inference/sample-terraform.tfvars:12]: Huh. Maybe my changes didn't work as expected?: https://github.com/GoogleCloudPlatform/ai-on-gke/blob/main/benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/run.sh#L24

I seem to remember testing both pathways. But maybe not

**Review by kfswain — APPROVED** [review]

File: benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/run.sh (hunk 2 above)
Comment (annapendleton) [reply benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/run.sh:42]: This is the output it gives around the existing comments:

"Token is valid (permission: read).
Your token has been saved to /root/.cache/huggingface/token"

I wanted additional logging as my previous issue wasn't showing any logs (even the login ones) so the echo commands were to identify what wasn't running.

File: benchmarks/benchmark/tools/locust-load-inference/sample-terraform.tfvars (hunk 1 above)
Comment (annapendleton) [reply benchmarks/benchmark/tools/locust-load-inference/sample-terraform.tfvars:12]: :)

**Comment by annapendleton** [comment]
tested end to end for http (manual testing required until we have presubmits).

**Pull request merged**